src/Tribe/Log/Service_Provider.php

```diff
@@ -22,6 +22,8 @@ class Service_Provider extends \tad_DI52_ServiceProvider {
 	 * @since 4.9.16
 	 */
 	public function register() {
+		$this->container->singleton( 'log', $this );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: c781657fcd16f457e41592fe0c8765f2</sub>